quinn-proto/src/config.rs

```diff
@@ -727,6 +730,17 @@ impl EndpointConfig {
         self.min_reset_interval = value;
         self
     }
+
+    /// Optional seed to be used internally for random number generation
+    ///
+    /// By default, quinn will initialize and endpoint's rng using a random seed. However, you can
```

Comment:
Typo, and "a random seed" is a bit confusing because it could be understood to describe what the data *is* instead of how it's populated.
```suggestion
    /// By default, quinn will initialize an endpoint's rng using a platform entropy source. However, you can
```

Reply:
That's much better wording, thanks! I amended the commit (instead of using GitHub's UI to accept your suggestion) to keep the git history clean.